src/ansys/mapdl/core/_commands/graphics/labeling.py

```diff
@@ -0,0 +1,1247 @@
+        :ref:`sbctran` ). Similarly, solid model plots do not show the load symbols if surface loads are
+        applied to nodes and elements. For node and element plots of shell element models, the surface load
+        symbols are shown only if the load face is visible from the current viewing direction.
+
+        The effects of the **/PSF** command are not cumulative (that is, the command does not modify an
+        existing setting from a
+        previously issued :ref:`psf` command). Only the setting specified via the most recent :ref:`psf`
+        command applies.
+
+        If you issue a `postprocessing (POST1)
+        <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en/ans_cmd/Hlp_C_CH2_7.html#cmdpost1failure>`_
+        plot command that produces result contours (such as :ref:`plnsol` ), :ref:`psf` has no effect. This
+        behavior prevents conflicting contours in the graphics window.
+
+        When using the radiosity method ( ``Item`` = RDSF and ``Comp`` = ENCL) with ``Key`` = 2, the
+        radiation arrows point outward from any element face.
+
+        :ref:`psf`,STAT displays current :ref:`psf` settings, and :ref:`psf`,DEFA resets them back to
+        default.
+
+        Other useful commands are :ref:`pnum`,SVAL,1 to show the values of the surface loads, :ref:`vscale`
+        to change arrow lengths, and :ref:`pbc` and :ref:`pbf` to activate other load symbols.
+
+        For beam elements, only the colors representing shear (GREEN) and normal (RED) pressures are
+        displayed for the arrows. The color of these arrows does not correspond to the magnitudes in the
+        contour legend. The length of these arrows does, however, correlate to the relative magnitude of the
+        pressures.
+
+        For elements SURF159, SOLID272, SOLID273, PIPE288 and PIPE289, :ref:`psf` is not available when
+        displaying elements with shapes determined from the real constants or section definition (
+        :ref:`eshape` ). For PIPE288 and PIPE289, only external loads applied via :ref:`sfbeam` are
+        displayed.
+
+        This command is valid in any processor.
+
+        .. _psf.tab.1:
+
+        **/PSF - Valid Item and Component Labels**
+
+        .. flat-table::
+           :header-rows: 1
+
+           * - Item
+             - Comp
+             - Description
+             - Comments
+           * - :rspan:`14` PRES [ :ref:`psf.labels.fn.1`]
+             - NORM (or blank)
+             - Applied pressure normal to the face (real component only).
+             - For element types other than  SURF153, SURF154 and SURF156.
+           * - NORM
+             - Applied pressure normal to the face (real component).
+             - :rspan:`5` For element types SURF153, SURF154 and SURF156 with KEYOPT(2) = 0. For `supported structural elements<https://ansyshelp.ansys.com/Views/Secured/corp/v232/en/ans_cmd/Hlp_C_SFCONTROL.html#>`_ with``KCSYS`` = 0. [ :ref:`psf.labels.fn.2`]
```

Comment:
Strange format on the table:

<img width="824" alt="image" src="https://github.com/user-attachments/assets/fc9ac6ce-9c84-490e-83c8-1d4014fbbfd0" />

Reply:
I tried different approaches in order to fix the link rendering but I have found the solution yet. I opened an [issue](https://github.com/ansys/pyconverter-xml2py/issues/416) to keep a track of it.